integration-tests/test-app/sub-mirror-input.ts

```diff
@@ -0,0 +1,34 @@
+import { registerHtml, useGlobalStore } from '../../src/tram-one';
+
+const html = registerHtml({
+	'sub-mirrorable-input': require('./sub-mirror-input'),
+});
+
+type InputObject = { value: string };
+
+/**
+ * component to test url parameters
+ */
+export default () => {
+	const mirrorable = useGlobalStore('mirrorable-input') as InputObject;
```

Comment:
The type isn't actually required if you pass in a default value. The fact that this option exists is cool, and possible because of function overloading.